src/views/News/index.js

```diff
@@ -297,10 +297,10 @@ export default class StudentNews extends Component {
     const networkStatus = JSON.parse(localStorage.getItem('network-status')) || 'online';
 
     if(this.props.Authentication) {
-
+      
       if (this.state.loading === true) {
         content = <GordonLoader />;
-      } else if (this.state.news.length > 0) {
+      } else if (this.state.news.length > 0 || this.state.personalUnapprovedNews.length > 0) {
```

Comment:
Shouldn't we check the length of this.state.filterednews instead of this,state,news? It seems that currently if filterednews was empty but there was some unexpired news, the news Card would just be empty and not say "No News to Show". Also, if we do decide to not check the length of 'news', there would be no need for this.state.news in the component and it could be removed.

Reply:
The addition looks good, though!

Reply:
@cpabbot can probably speak better to this then I can, but it looks like currently, filtering news is disabled, so news and filtered news will have the same value. However, when search is implemented, we will need to think of what to display in the case that there is no news matching the result.

Probably good to proactively update this if to check filterednews as you suggested Josh, good catch!